draft-irtf-cfrg-hpke.md

```diff
@@ -481,6 +481,10 @@ information alongside a ciphertext from sender to receiver. Specification of
 such a mechanism is left to the application. See {{message-encoding}} for more
 details.
 
+Note that some KEMs may not support the AuthEncap or AuthDecap functions.
+For such KEMs, only mode_base or mode_psk may be supported. Future specifications
```

Comment:
Nit: "may" is not the right verb.  Maybe "are".

Reply:
That's better -- thanks.